lib/Basic/LangOptions.cpp

```diff
@@ -264,6 +265,13 @@ std::pair<bool, bool> LangOptions::setTarget(llvm::Triple triple) {
   // Set the "os" platform condition.
   switch (Target.getOS()) {
   case llvm::Triple::Darwin:
+    addPlatformConditionValue(PlatformConditionKind::OS, "Darwin");
```

Comment:
Would it be possible though? Because we're aliasing one term (Darwin) to check for 4 others, but if we could alias that would be *way* better

Reply:
You should be able to recurse to check multiple different aliases in `LangOptions::checkPlatformCondition`, similar to the existing aliases, but you check for multiple and bail if you get any match (ideally it would be nice if it were refactored into a predicate taking function, though I don't think `PlatformConditionValues` is expected to be large).

Reply:
Think I figured it out, see https://github.com/apple/swift/pull/59426/commits/3c21246c7e01de5d3cb82ec89faec6a04e90a0e6

Reply:
Looks good to me, this change here can now be removed right?

Reply:
Oh yeah, about that, must we not add `addPlatformConditionValue(PlatformConditionKind::OS, "Darwin");` under iOS, macOS, tvOS and watchOS in that switch?

Reply:
I see, i'll remove it then